clar2wasm/src/words/sequences.rs

```diff
@@ -1753,17 +1769,28 @@ mod tests {
   (list 1 2 3 4)
   (list 10 20 30))
 ";
-        assert_eq!(clarity::vm::execute(a).unwrap(), eval(a));
+        assert_eq!(execute(a).unwrap(), eval(a));
     }
 
     #[test]
-    fn test_builtin_2() {
+    fn map_and() {
         let a = "
-(map +
-  (list 1 2 3 4)
-  (list 10 20 30)
-  (list 100 200 300))
+(map and
+  (list true true true)
+  (list false true true)
+  (list false false true))
+";
+        assert_eq!(execute(a).unwrap(), eval(a));
+    }
+
+    #[test]
+    fn map_or() {
+        let a = "
+(map and
```

Comment:
```suggestion
(map or
```
Just noticed this and removed the PR from queue. That explains why the codecov dropped so much. I should've investigated that before I approved 🙈.